src/containers/Accounts/AccountHeader/index.js

```diff
@@ -170,7 +170,8 @@ class AccountHeader extends Component {
             <li key={xAddress.classicAddress}>
               <span className="classicAddress">
                 <span className="label">Classic Address: </span>
-                <Account account={xAddress.classicAddress} /> {/* TODO: Add [copy] button */}
+                <Account account={xAddress.classicAddress} parentPath={parentPath} />{' '}
+                {/* TODO: Add [copy] button */}
```

Comment:
what is this?

Reply:
Not sure - it was there before, the linter just moved it on save